cilium-health/launch/endpoint.go

```diff
@@ -18,6 +18,7 @@ import (
 	"bufio"
 	"context"
 	"fmt"
+	"net"
```

Comment:
The commit messages doesn't describe the actual changes.
> cilium-health: remove direct dependency on pkg/k8s

How can this be possible if there is no import removal of a `pkg/k8s`?